src/function_lua.c

```diff
@@ -476,7 +476,7 @@ int luaEngineInitEngine() {
                             "    i = dbg.getinfo(3,'nSl')\n"
                             "  end\n"
                             "  if i then\n"
-                            "    return i.source .. ':' .. i.currentline .. ': ' .. err\n"
+                            "    return err .. '. ' .. i.source .. ':' .. i.currentline\n"
```

Comment:
Lets try to use this PR to unify this with the `errh_func` script in `scriptingInit()`.